example/server/server.ts

```diff
@@ -607,10 +734,73 @@ function handleMediaEvent(event: TemplateEvent<ServerAction>): boolean {
     })
     return true
   }
+  if (action === 'removeItem') {
+    const targetPath = mediaPath.slice(0, -2)
+    const itemKey = event.action.at(-1)
+    rootMediaUpdate(targetPath, (media) => {
+      if (media.type !== 'sequence') return media
+      return {
+        ...media,
+        sequence: (media.sequence || []).filter((item) => item.key !== itemKey),
+      }
+    })
+    return true
+  }
+  if (action === 'maxDuration') {
+    let duration = event.payload
+    if (Array.isArray(duration)) duration = duration[0] // slider gives us an array
+    if (duration === true) duration = 10
```

Comment:
Can duration be a boolean if sliders give an array?

Reply:
Also, I think we can safely just write `event.payload[0]`. At least that's what I've been doing in other places where this event is expected to come from the Slider. Part of its future signature I guess.